.gitlab-ci.yml

```diff
@@ -72,8 +73,8 @@ test:acceptance:
 
 test:acceptance:enterprise:
   extends: .template:test:acceptance
-  only:
-    - master
+  # only:
+  #   - master
```

Comment:
Is this intentional?

Reply:
nope - thanks for catching it! that was needed for the evaluation (the tests + corresponding coverage collection should still only run on master)